### DISCUSSION TITLE
Front-End tests

### DISCUSSION THREAD

**Discussion by patrykgz** [ant-design-blazor/ant-design-blazor#1233] [Ideas]
We should make a new nuget package for testing AntDesign-based apps!

**Comment by ElderJames** [comment]
That's a good suggestion. I would help to publish it to nuget server.

**Reply by patrykgz** [reply]
Should we package AntDesign.Tests or make new csproj?

**Reply by ElderJames** [reply]
How about creating a new project called AntDesign.TestKit?

**Comment by patrykgz** [comment]
#1248
